src/styles/blocks/content.scss

```diff
@@ -0,0 +1,8 @@
+.content {
+  &__watch {
+    height: 100vh;
```

Comment:
The height should be 80vmin according to the task description. Change '100vh' to '80vmin'.